lib/misc/cells.js

```diff
@@ -0,0 +1,64 @@
+(function() {
+  var scopes, values;
+
+  scopes = require('./scopes');
+
+  values = require('underscore-plus').values;
```

Comment:
This is kinda unidiomatic -- should be `import { values } from 'underscore-plus'` at the toplevel. Same above.